prime-router/src/main/kotlin/azure/Event.kt

```diff
@@ -16,6 +16,7 @@ abstract class Event(val action: Action, val at: OffsetDateTime?) {
     abstract fun toQueueMessage(): String
 
     enum class Action {
+        RECEIVE,
         TRANSLATE, // Deprecated
```

Comment:
The rule is that every Report is created by an Action, even the initially submitted report.   So I had to create an 'receive' action to cover that situation.